packages/workspace/src/utils/json-diff.spec.ts

```diff
@@ -33,6 +33,30 @@ describe('jsonDiff', () => {
     );
   });
 
+  it('should have diffs for objects as well', () => {
```

Comment:
Maybe we can add some tests for longer arrays and also to stress that if a granch child changes, child, parent will change as well

Reply:
Added tests.